vaccine_feed_ingest/runners/ma/ma_immunizations/fetch.yml

```diff
@@ -0,0 +1,9 @@
+---
+state: ma
+site: ma_immunizations
+parser: prepmod
+url: https://www.maimmunizations.org
+params:
+  q[venue_search_name_or_venue_name_i_cont]: ''
+headers:
+  User-Agent: github.com/CAVaccineInventory
```

Comment:
Could you use `Vaccinebot (+https://www.vaccinatethestates.com/vaccinebot)` as the user agent?

We'll be adding a page to vaccinateTheStates.com similar to [this page on vaccinateCA.com](https://www.vaccinateca.com/vaccinebot) to provide more information for folks who look it up.

Reply:
Yep, done.